Source/YOChartImageKit/YOBarChartImage.h

```diff
@@ -30,6 +30,12 @@ typedef NS_ENUM(NSInteger, YOBarChartImageBarStyle){
 @property (nonnull) NSArray<NSNumber *> *values;
 
 /**
+ *  The maximum value to use for the chart. Setting this will override the 
+ *  default behavior of using the highest value as maxValue.
+ */
+@property NSNumber* maxValue;
```

Comment:
Could you add `(nonnull, nonatomic)`?

I've added `(nonatomic)` declaration.
https://github.com/yasuoza/YOChartImageKit/blob/master/Source/YOChartImageKit/YOBarChartImage.h#L30